benchwise/models.py

```
@@ -24,7 +27,125 @@ def get_cost_estimate(self, input_tokens: int, output_tokens: int) -> float:
+            for i, prompt in enumerate(batch):
+                task = process_single_prompt(batch_start_idx + i, prompt)
+                all_tasks.append(task)
+        
+        # Gather all results with exception handling
+        results_with_indices = await asyncio.gather(*all_tasks, return_exceptions=True)
+        
+        # Handle any exceptions that weren't caught
+        processed_results = []
+        for item in results_with_indices:
+            if isinstance(item, Exception):
+                processed_results.append((len(processed_results), f"Error: {str(item)}"))
```

Ruff (0.14.8)

Use explicit conversion flag

Replace with conversion flag

(RUF010)